### PR TITLE
beautiful piece of code here

### PR DESCRIPTION
**Pull request by OhRobin**
This piece of code is so beautiful that when I look at it directly, it makes me go blind.

### CODE REVIEW

File: HW02/HW02_ch03_ex01.py [hunk 1]
```diff
@@ -15,10 +15,9 @@
 # Write your function below:
 # Body
 
-
-
-
-
+def right_justify(s):
+	x = len(s)
+	print(" "*(70-x) + s)
```
Comment (danielsgriffin) [inline HW02/HW02_ch03_ex01.py]: Good.

Comment (danielsgriffin) [inline HW02/HW02_ch03_ex01.py]: Are these tabs or spaces? Please consider setting your text-editor to convert to tabs to four spaces.

**Comment by danielsgriffin** [comment]
Great job.
What text editor are you using?
Have you ever used a linter? Checkout something like this: https://github.com/SublimeLinter/SublimeLinter-flake8

File: HW02/HW02_ch03_ex01.py (hunk 1 above)
Comment (OhRobin) [inline HW02/HW02_ch03_ex01.py]: https://www.youtube.com/watch?v=SsoOG6ZeyUI

**Comment by OhRobin** [comment]
I'm using Sublime. I just changed user preference to to convert tabs to spaces

**Pull request closed without merging**